KEYLESS.md

```diff
@@ -114,14 +112,12 @@ Signature timestamps are checked in the [rekor](https://github.com/sigstore/reko
 ## Upcoming work
 
 * Root CA hardening: We should use intermediate certs rather than the root, and support chained verification.
-* Root CA configuration: We should allow users to change the roots and add their own.
```

Comment:
I think the bullet point above is also fixed? @haydentherapper now that we use Fulcio intermediates for the public good instance & support cert chains (I think this is what this meant)